ophyd/quadem.py

```diff
@@ -22,6 +22,8 @@ class QuadEM(SingleTrigger, DetectorBase):
     _default_configuration_attrs = ('integration_time', 'averaging_time')
     _default_read_attrs = ('current1.mean_value', 'current2.mean_value',
                            'current3.mean_value', 'current4.mean_value')
+    _default_hints = {'fields': ['current1', 'current2',
```

Comment:
Looking at the deleted block, I see that the original default hints had the form `self.current1.mean_value.name`. Therefore, these entries should have the form `'current1.mean_value'`.